docs/Lifecycle/DefinitionOfDone.md

```diff
@@ -0,0 +1,115 @@
+# Definition of Done (DoD)
+According SDL Contribution result of implementation should b a Pull Request to one or many SDL repositories.<br>
+Every Pull Requests should have following description:<br>
+> **Pull requests** *\<other pull requests which contain additional deliverables, if applicable\>*<br>
+> - Pull request 1<br>
+> - Pull request 2<br>
+> ...<br>
+> **Tasks** *\<implemented technical tasks/defects\>*<br>
+> - Technical task 1<br>
+> *Deliverables:*<br>
+> [ ] Deliverable 1<br>
+> [ ] Deliverable 2<br>
+> [ ] Deliverable 3<br>
+> ...<br>
+> - Technical task 2<br>
+> *Deliverables:*<br>
+> [ ] Deliverable 1<br>
+> [ ] Deliverable 2<br>
+> [ ] Deliverable 3<br>
+> ...<br>
+> - Defect 1<br>
+> *Deliverables:*<br>
+> [ ] Deliverable 1<br>
+> [ ] Deliverable 2<br>
+> [ ] Deliverable 3<br>
+> - Defect 2<br>
+> *Deliverables:*<br>
+> [ ] Deliverable 1<br>
+> [ ] Deliverable 2<br>
+> [ ] Deliverable 3<br>
+> ...<br>
+
+Following is the checklist of activities that add verifiable/demonstrable value to the product.<br>
+Each and every delivery to SDL should be verified for readiness upon this checklist.<br>
+DoD is not static; it evolves with SDL.<br>
+It is required to use this document as a checklist each time preparing delivery.<br>
+
+## Delivery completeness checklist
+Complete delivery to SDL consist of following deliverables:
+1. Source codes
+2. Code comments
+3. Unit tests (UTs)
+4. Integration tests (for [ATF][ATF-LINK])
+5. SDD/SAD updates
+6. Other guidelines
+
+Technical tasks must contains in its description explicit list of stated items.<br>
+In case of trivial changes which do not require SDD/SAD/Guidelines updates,<br>
+technical task might contained shrtened list of deliverables.<br>
+In this case only stated in technial task expected delivery have to be implemented.<br>
+
+### Source codes
```

Comment:
I thin we can reorgonize this and following few chapters with information about code with links to SAD
https://smartdevicelink.com/en/guides/core/software-architecture-document/development-view/#481-implementation-technologies
https://smartdevicelink.com/en/guides/core/software-architecture-document/development-view/#483-development-environment-and-standards
I don't see any need to update this information in more than one place.

If you want we can rearrange these chapters in SAD.